templates/store/snap-details/_channel_map.html

```diff
@@ -61,6 +71,9 @@
               </div>
             </div>
           </form>
+          {% if not has_stable %}
+          <p><i class="p-icon--warning"></i> {{ snap_title }} is only available at the unstable {{lowest_risk_available}} channel. It could break and change often.</p>
```

Comment:
Is is not better to use "is only available **on** the unstable" instead? Would be consistent with https://github.com/canonical-web-and-design/snapcraft.io/pull/2897/files#diff-2a6fe6590c20bc2970144752c7920138R327